internal/backends/qt/qt_window.rs

```diff
@@ -1532,7 +1532,18 @@ impl WindowAdapterSealed for QtWindow {
                 size.height = existing_size.height;
             }
         }
-        let background: u32 = window_item.background().as_argb_encoded();
+
+        let window_background_brush =
+            window_inner.window_item().map(|w| w.as_pin_ref().background());
```

Comment:
we already have window_item in a local variable, so no need to call map and stuff.
Also with Qt we can just transform the background brush in a QBrush (there should already have a helper function for that) and set that as a palette for the window background

Reply:
done thank you